claragenomics/dl4atac/models/models.py

```diff
@@ -7,81 +7,89 @@
 # distribution of this software and related documentation without an express
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
+"""Model definitions."""
+
+from claragenomics.dl4atac.layers import ConvAct1d, DownBlock, \
+    ResBlock, UpBlock, ZeroSamePad1d
 
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
-from claragenomics.dl4atac.layers import ZeroSamePad1d, Activation, ConvAct1d, ResBlock, DownBlock, UpBlock
-
-
-class FC3(nn.Module):
-    def __init__(self, interval_size, in_channels=1, afunc='relu', bn=False):
-        self.interval_size = interval_size
-        super(FC3, self).__init__()
-
-        self.fc1 = nn.Linear(in_features=interval_size,
-                             out_features=1000)
-        self.relu1 = Activation('relu')
-        self.fc2 = nn.Linear(in_features=1000,
-                             out_features=1000)
-        self.relu2 = Activation('relu')
-        self.fc3 = nn.Linear(in_features=1000, out_features=interval_size)
-
-    def forward(self, x):
-        x = self.relu1(self.fc1(x))
-        x = self.relu2(self.fc2(x))
-        x = self.fc3(x)
-        x = torch.sigmoid(x)
-        return x.squeeze(1)
-
-
-class FC2(nn.Module):
-    def __init__(self, interval_size, in_channels=1, afunc='relu', bn=False):
-        self.interval_size = interval_size
-        super(FC2, self).__init__()
-
-        self.fc1 = nn.Linear(in_features=interval_size,
-                             out_features=1000)
-        self.relu1 = Activation('relu')
-
-        self.fc2 = nn.Linear(in_features=1000, out_features=interval_size)
-
-    def forward(self, x):
-        x = self.relu1(self.fc1(x))
-        x = torch.sigmoid(self.fc2(x))
-        return x.squeeze(1)
 
 
 class DenoisingResNet(nn.Module):
-    def __init__(self, interval_size, in_channels=1, out_channels=15, num_blocks=5,
-                 kernel_size=50, dilation=8, bn=False, afunc='relu', num_blocks_class=2,
-                 kernel_size_class=50, dilation_class=8, out_channels_class=15):
-
+    """Resnet model."""
+
+    def __init__(self, interval_size, in_channels=1, out_channels=15,
+                 num_blocks=5,
+                 kernel_size=50, dilation=8, bn=False, afunc='relu',
+                 num_blocks_class=2,
+                 kernel_size_class=50, dilation_class=8,
+                 out_channels_class=15):
+        """Initialize the class.
+
+        Args:
+            interval_size : length (in bp) of the genomic intervals supplied
+            to the model.
+            in_channels : Number of input channels.
+            out_channels: Number of output chanels.
+            num_blocks: Number of blocks.
+            kernel_size: Size of the kernel filter.
+            dilation: dilation parameter.
+            bn: Batch normalization.
+            afunc: activation function.
+            num_blocks_class:
+            kernel_size_class:
```

Comment:
Size of the kernel filter for all convolutional layers in the classification part of the model